src/edu/csus/ecs/pc2/core/execute/ExecuteTimerFrame.java

```diff
@@ -0,0 +1,237 @@
+     */
+    private JPanel getJFrameContentPane() {
+        if (ivjJFrameContentPane == null) {
+            try {
+                ivjJFrameContentPane = new javax.swing.JPanel();
+                ivjJFrameContentPane.setName("JFrameContentPane");
+                ivjJFrameContentPane
+                        .setLayout(getJFrameContentPaneBorderLayout());
+                getJFrameContentPane().add(getExecuteTimerLabel(), "North");
+                getJFrameContentPane().add(getTimerCountLabel(), "Center");
+                getJFrameContentPane().add(getbtnTerminate(), "South");
+            } catch (java.lang.Throwable ivjExc) {
+                handleException(ivjExc);
+            }
+        }
+        return ivjJFrameContentPane;
+    }
+
+    /**
+     * Return the JFrameContentPaneBorderLayout property value.
+     *
+     * @return java.awt.BorderLayout
+     */
+    private BorderLayout getJFrameContentPaneBorderLayout() {
+        java.awt.BorderLayout contentPaneBorderLayout = null;
+        try {
+            /* Create part */
+            contentPaneBorderLayout = new java.awt.BorderLayout();
+            contentPaneBorderLayout.setVgap(10);
+        } catch (java.lang.Throwable ivjExc) {
+            handleException(ivjExc);
+        }
+
+        return contentPaneBorderLayout;
+    }
+
+    public void resetFrame()
+    {
+        if(ivjExecuteTimerLabel != null) {
+            ivjExecuteTimerLabel.setText("Execution Time");
+            ivjExecuteTimerLabel
+                    .setVerticalTextPosition(javax.swing.SwingConstants.BOTTOM);
+            ivjExecuteTimerLabel
+                    .setFont(new java.awt.Font("dialog", 1, 18));
+            ivjExecuteTimerLabel
+                    .setVerticalAlignment(javax.swing.SwingConstants.BOTTOM);
+            ivjExecuteTimerLabel
+                    .setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+            ivjExecuteTimerLabel.setCursor(new java.awt.Cursor(
+                    java.awt.Cursor.DEFAULT_CURSOR));
+        }
+        if(ivjTimerCount != null) {
+            ivjTimerCount.setForeground(java.awt.Color.black);
+            ivjTimerCount.setFont(new java.awt.Font("monospaced", 1, 48));
+            ivjTimerCount.setText(" ");
+            ivjTimerCount
+                    .setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+            ivjTimerCount.setCursor(new java.awt.Cursor(
+                    java.awt.Cursor.DEFAULT_CURSOR));
+        }
+        if(ivjbtnTerminate != null) {
+            ivjbtnTerminate.setMnemonic('t');
+            ivjbtnTerminate.setText("Terminate");           
+        }
+    }
+    
+    public void setTimerFrameVisible(boolean bVis)
+    {
+        setVisible(bVis);
+    }
+    
+    /**
+     * Return the ExecuteTimerLabel1 property value.
+     *
+     * @return javax.swing.JLabel
+     */
+    private JLabel getExecuteTimerLabel() {
+        if (ivjExecuteTimerLabel == null) {
+            try {
```

Comment:
The contents of this `try` block appear to be identical to the settings applied in the first `if` statement in method `resetFrame()`, above.  It seems like having two separate places setting the same values is just inviting a down-the-road issue where one of the places gets changed but the other does not, resulting in inconsistent GUI appearance/behavior.  It seems like the two code blocks should be encapsulated in a single method...

Reply:
Agreed.

Reply:
I sort of interpreted "Agreed" as meaning "I agree, and I'll fix it".  But I don't see any changes.  Are you agreeing that it should be changed but declining to change it?

Reply:
This has been fixed and pushed.